Automattic/jetpack
# ThreatsDataViews: Add ToggleGroupControl

## Description
Adds a toggle control the `ThreatsDataViews` component header to allow for preset filters (easily toggling between active and historic threats).

## Proposed changes:

* Applies a custom `ToggleGroupControl` component to the header of the `ThreatsDataViews`

### Other information:


## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1477

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch
* Load storybook with cd projects/js-packages/storybook && npm run storybook:dev
* View the Threats Data View component
* Ensure the `ToggleGroupControl` header content is positioned and styled according to designs in both Table and List views and that the filter are appropriately applied and unapplied when expected.
* Review the code and ensure the implementation is correct

## Screenshots
![Screen Shot on 2024-11-04 at 13-51-59](https://github.com/user-attachments/assets/bfd4669b-a4ab-43c6-9545-2edde8e30b0c)
